src/database/sql/sequelize/mappers/educational/learning/learning.path.mapper.ts

```diff
@@ -17,9 +17,10 @@ export class LearningPathMapper {
             Description      : course.Description,
             ImageUrl         : course.ImageUrl,
             DurationInDays   : course.DurationInDays,
-            Courses          : [],
+            Courses          : course.LearningCourses,
```

Comment:
When we are listing courses under learning journey, are we also adding sequence of them in the DTO?